### ISSUE TITLE
Profile

### ISSUE THREAD

**Issue by smadamit** [inbaz/ers#57]
feature missing

Profile	index
Profile	edit
Profile	password
Profile	forgotPassword
Profile	package
Profile	participant

Profile: password reset
Profile: resend email

**Comment by smadamit** [comment]
same issues: #136 and #137

**Comment by inbaz** [comment]
Yeah, password reset is working now, checkout /profile/request-password in 0.2.35

**Comment by smadamit** [comment]
0.2.36:

tesed: /profile/request-password
- If I follow the link in the email I end here: user/login So I can't reset my password.

**Comment by inbaz** [comment]
Yep, wrong operator used. This is fixed in 0.2.37

**Comment by inbaz** [comment]
I already added some features in 0.3.22, please check the functions and create a list which profile functions are really needed this year.

**Comment by smadamit** [comment]
features implemented:
- admin panel -> change user data
- user -> change password -> tested in 0.3.23 -> OK (error message for wrong old password: change text and color:red)
- user -> change user data -> tested in 0.3.23 -> OK

**Comment by smadamit** [comment]
first test results in 0.3.32:

- #378
- #379 
- #380 
- #381
